index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
 
 <h2>Calls for implementation and feedback</h2>
 
-<p>When an addition is accepted at stage 3, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage, and external feedback.
+<p>When an addition is accepted at Stage 3, the committee is signifying that design work is complete and further refinement will require implementation experience, significant usage, and external feedback.
 
 <h2>Tips for achieving consensus</h2>
 
-<p>During the discussion of a proposal any aspect may be discussed. Consensus is given as an indicator of the current stage. Delegates should openly give feedback on proposals, and especially for a proposal for stage advancement where the concern is relevant to the stage. Delegates should raise their concerns early and asynchronously, in order to help the champion resolve any issues.
+<p>During the discussion of a proposal, any aspect may be addressed. Delegates are encouraged to provide feedback on proposals, especially on stage advancement proposals where concerns are relevant to that stage. Feedback and concerns should be raised early and asynchronously to help the champion resolve any issues efficiently. Consensus determines whether a proposal is ready to advance to the next stage.
 
-<p>A delegate may pose a constraint as necessary for advancement. A constraint refers to an desired property of the proposal, accompanied by a rationale. We encourage this to also be done asynchronously in issues, and in incubator calls, as well as in plenary. In this situation, the delegate should expect to work with the champion and other delegates during earlier stages to incorporate their constraint into the solution, and to consider different possible tradeoffs. In general, the earlier a constraint is raised, the better.
+<p>A delegate may pose a constraint as necessary for advancement. A constraint refers to a desired property of the proposal, supported by a rationale. We encourage this to be done asynchronously in issues, in incubator calls, and in plenary. In this situation, the delegate should work with the champion and other delegates during earlier stages to incorporate the constraint into the solution, and to explore possible tradeoffs. Generally, the earlier a constraint is raised, the better.
 
-<p>Frequently, many different conflicting constraints are posited about proposals, and the committee collectively may make tradeoffs selecting a particular design even though it compromises one or more constraints.
+<p>Often, multiple conflicting constraints are proposed, and the committee may make tradeoffs, selecting a design that compromises one or more constraints.
```

Comment:
Attempting to reserve "propose"/"proposal" for the top-level concept:
```suggestion
<p>Often, multiple conflicting constraints are raised, and the committee may make tradeoffs, selecting a design that compromises one or more constraints.
```